src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -180,12 +182,30 @@ export class DotEditContentHtmlService {
      * @param {ContentType} form
      * @memberof DotEditContentHtmlService
      */
-    renderAddedForm(form: ContentType): void {
-        this.renderAddedItem({
-            item: form,
-            checkExistFunc: this.isFormExistInContainer.bind(this),
-            getContent: this.dotContainerContentletService.getFormToContainer.bind(this.dotContainerContentletService)
-        });
+    renderAddedForm(form: ContentType): Observable<DotPageContainer[]> {
+        const doc = this.getEditPageDocument();
+        const containerEl = doc.querySelector(
+            // tslint:disable-next-line:max-line-length
```

Comment:
If you use an `[]` and then `.join` you don't need to disable the `max-line-lenght` and is more readable.